models/surveys.go

```diff
@@ -387,6 +453,12 @@ func (api *API) PostSurveyClassifiers(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	surveyID := vars["surveyId"]
 
+	isValid := IsValidUUID(surveyID)
+	if isValid == false {
```

Comment:
🎨 but Go would accept `if !isValid` - the only reason to not do this would be if `isValid` could be `null` as `isValid == false` has built in null-checking (what a world!)

Reply:
oh yeah, good point.  I'll take a look at doing that